src/jekyll/_plugins/custom_tags/audio.rb

```diff
@@ -0,0 +1,59 @@
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
Is there a reason why you did this as an audio tag vs an include? Personally I prefer includes, but not set absolutely.